content/documentation/glossary/observability/index.adoc

```diff
@@ -0,0 +1,40 @@
+---
+title: "Observability"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Observability Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Observability Glossary Content
+:keywords: Kiali Documentation Glossary
+:icons: font
+
+== Citadel
+
+For more information see the <<Citadel>> definition in link:https://istio.io/docs/concepts/what-is-istio/#citadel[istio Citadel].
+
+== Envoy
+
+A proxy that Istio starts for each pod in the service mesh.
+For more information see the <<Envoy>> definition in link:https://istio.io/docs/concepts/what-is-istio/#envoy[Istio Envoy].
```

Comment:
For more information see the [Envoy] definition in the link:https://istio.io/docs/concepts/what-is-istio/#envoy[Istio Envoy documentation].